samples/nrf9160/coap_client/src/main.c

```diff
@@ -92,115 +86,98 @@ static int client_init(void)
 {
 	int err;
 
-	const struct sockaddr_in client_addr = {
-		.sin_port = 0,
-		.sin_family = AF_INET,
-		.sin_addr.s_addr = 0
-	};
-
-	coap_local_t local_port_list[] = {
-		{
-			.addr = (struct sockaddr *)&client_addr,
-			.protocol = IPPROTO_UDP,
-			.setting = NULL
-		}
-	};
-
-	coap_transport_init_t transport_param = {
-		.port_table = local_port_list
-	};
-
-	err = coap_init(sys_rand32_get(), &transport_param, k_malloc, k_free);
-	if (err != 0) {
-		printf("Failed to initialize CoAP %d\n", err);
-		return -err;
+	sock = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
+	if (sock < 0) {
+		printk("Failed to create CoAP socket: %d.\n", errno);
+		return -errno;
 	}
 
-	/* NOTE: transport_handle is the socket descriptor. */
-	transport_handle = local_port_list[0].transport;
+	err = connect(sock, (struct sockaddr *)&server,
+		      sizeof(struct sockaddr_in));
+	if (err < 0) {
+		printk("Connect failed : %d", errno);
```

Comment:
Missing newline

Reply:
Where exactly?

Reply:
Was thinking about `\n` at the end of the printk message

Reply:
Ah right, will fix, thanks!